src/eonstor_ds_cli/common_cli.py

```diff
@@ -232,6 +220,29 @@ def _init_pool_list(self):
         tmp_pool_list = pools_name.split(',')
         self.pool_list = [pool.strip() for pool in tmp_pool_list]
 
+        """Record the tier pools information.
+
+        tier_pools_dict = {
+            'LV0': [0, 1, 2, 3], # LV0 has 4 tiers which are 0, 1, 2, 3
+            'LV1': [0, 1, 3]     # LV1 has 3 tiers which are 0, 1, 3
+        }
+        """
+
+        rc, pools_info = self._execute('ShowLV')
+        rc, lv_info = self._execute('ShowLV', 'tier')
```

Comment:
'ShowLV'' 'tier' will show tier pools only. Therefore, it's necessary to call showlv twice.